02Lesson/01Homework/index.html

```diff
@@ -1,8 +1,54 @@
 <!DOCTYPE html>
-<html>
-<head>
-  <title></title>
-</head>
-<body>
-</body>
+<html style= "background: lightblue" >
+  <head>
+    <title>Jonathan Gleason</title>
+  </head>
+  <body>
+    <h1>Jonathan Gleason
+    </h1>
+    <h5><em>2101 Burton Dr. Austin, TX 78741</em>
+    </h5>
+    <h2>Background</h2>
+    <p>
+      I am a 25 year old entrepreneur who recently moved to Austin, TX in order to pursue my
+      dream of starting my own business. Before coming to Austin, I spent two years in Dallas, TX
+      working as a financial analyst at MassMutual. The passion for technology became more apparent during my
+      college years at the University of Georgia, but my dream to start my own company has always been in me
+      since a little boy.
+    </p>
+
+    <h3>Hobbies
+    </h3>
+
+      <ul>
```

Comment:
Indentation: After closing element, the next element needs to be on the same level.